README.md

```diff
@@ -59,7 +59,7 @@
 | SITE_MAIL  | Site e-mail address | admin@example.com |
 | ADMIN_NAME  | Admin username | admin |
 | PROJECT_INSTALL | Way to install site - from straight or existing config | - |
-| IMAGE_PHP | Php image to use | `skilldlabs/php:81-fpm` |
+| IMAGE_PHP | Php image to use | `skilldlabs/php:82-unit` |
```

Comment:
I pushed the change as 94a071e5ac1e66988e65968f1837c9f03519755b

Reply:
please rebase and squash commits

Reply:
Rebased, squashed @andypost

Reply:
Thank you!